gs-web-elasticsearch/doc/index.rst

```diff
@@ -296,60 +296,170 @@ Grid Strategy
 ^^^^^^^^^^^^^
 The ``gridStrategy`` parameter identifies the ``mil.nga.giat.process.elasticsearch.GeoHashGrid`` implemenation that will be used to convert each geohashgrid bucket into a raster value (number).
 
-* ``basic``: returns top level ``doc_count``.
+Basic
+~~~~~
+Returns top level ``doc_count``.
 
-    Example Aggregation::
+Example Aggregation::
+
+  {
+    "agg": {
+      "geohash_grid": {
+        "field": "geo"
+      }
+    }
+  }
     
-      {
-        "agg": {
-          "geohash_grid": {
-            "field": "geo"
+Example bucket::
+
+ {
+   "key" : "xv",
+   "doc_count" : 1
+ }
+
+Extracted raster value: ``1``
+
+Metric
+~~~~~~
+Returns metric value.
+
+.. list-table::
+   :widths: 20 20 60
+
+   * - Parameter
+     - Default Value
+     - Description
+   * - Metric key
+     - ``metric``
+     - Key used to pluck metric object from top level bucket. Empty string results in plucking doc_count.
+   * - Value key
+     - ``value``
+     - Key used to pluck the value from the metric object.
+
+Example Aggregation::
+
+  {
+    "agg": {
+      "geohash_grid": {
+        "field": "geo"
+      },
+      "aggs": {
+        "metric": {
+          "max": {
+            "field": "magnitude"
           }
         }
       }
+    }
+  }
+
+Example bucket::
+
+  {
+    "key" : "xv",
+    "doc_count" : 1,
+    "metric" : {
+      "value" : 4.9
+    }
+  }
     
-   Example bucket::
- 
-     {
-       "key" : "xv",
-       "doc_count" : 1
-     }
-  
-  Raster value: ``1``
-
-* ``metric``: returns metric value.
-
-    Example Aggregation::
-    
-      {
-        "agg": {
-          "geohash_grid": {
-            "field": "geo"
-          },
-          "aggs": {
-            "metric": {
-              "max": {
-                "field": "magnitude"
-              }
-            }
+Extracted raster value: ``4.9``
+
+Nested_agg
+~~~~~~~~~~

```

Comment:
How about creating a table in the top level Grid Strategy section above specifying the three strategies and include the actual required `grid_strategy` value (e.g. `nested_agg`) there? Then you could refer to that as just "Nested" here which I think would be a little cleaner.